packages/storybook8/stories/Components/ImageOverlay/index.stories.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ImageOverlay as ImageOverlayComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+
+import { ImageOverlayExample } from './snippets/ImageOverlay.snippet';
+import { controlsToAdd, hiddenControl } from './utils';
+
+export { Preview } from './ImageOverlay.story';
```

Comment:
Where is the Preview being used?

Reply:
Updated the name to be ImageOverlay instead of Preview. Good catch